code/modules/projectiles/updated_projectiles/ammunition.dm

```diff
@@ -505,7 +505,13 @@ Turn() or Shift() as there is virtually no overhead. ~N
 
 
 
-
+/obj/item/ammo_magazine/fortyFive
```

Comment:
```suggestion
/obj/item/ammo_magazine/acp
```
don't use camelcase for naming typepaths

Reply:
New habit.